templates/plugin/dhcp_remote_isc.yml.erb

```diff
@@ -0,0 +1,30 @@
+---
+#
+# Configuration file for remote ISC dhcp provider
+#
+
+:config: <%= scope.lookupvar("::foreman_proxy::plugin::dhcp::remote_isc::dhcp_config") %>
+:leases: <%= scope.lookupvar("::foreman_proxy::plugin::dhcp::remote_isc::dhcp_leases") %>
+
+# Redhat 5
```

Comment:
We have these in the template for the core dhcp isc provider, which is the reason I kept them here. I have no preference.